requirements-dev.in

```diff
@@ -0,0 +1,7 @@
+ipython
```

Comment:
I like putting these in alphabetical order, so that it is clear where to add new packages later, when such . aneed arises. not obligatory in any way though. 🙂

Reply:
BTW, isn't `ipython` a good addition for a dev env, but a redundant embellishment for a prod env?

Reply:
Yeah, actually it was supposed to be only in dev requirements, good catch!

Reply:
oh, wait it is dev requirements, nevermind

Reply:
ah 😅 doppelganger

Reply:
yeah 😄

And about alphabetical order, I've used to add new requirements at the end, so from the file you'll see what order the requirements were added to the project. That being said, I don't think I've ever really needed that order for anything, so I might as well try alphabetical ordering on this project 😄